galley/cmd/galley/cmd/probe.go

```diff
@@ -30,15 +30,15 @@ func probeCmd() *cobra.Command {
 	prb := &cobra.Command{
 		Use:   "probe",
 		Short: "Check the liveness or readiness of a locally-running server",
-		Run: func(cmd *cobra.Command, _ []string) {
+		RunE: func(cmd *cobra.Command, _ []string) error {
 			if !probeOptions.IsValid() {
-				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "some options are not valid")
-				return
+				return fmt.Errorf("some options are not valid")
 			}
 			if err := probe.NewFileClient(&probeOptions).GetStatus(); err != nil {
-				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "fail on inspecting path %s: %v", probeOptions.Path, err)
+				return fmt.Errorf("fail on inspecting path %s: %v", probeOptions.Path, err)
```

Comment:
Would it make sense to create a unique Error struct and associate it with a specific code? e.g.

https://github.com/istio/istio/blob/master/istioctl/cmd/analyze.go#L37-L54
https://github.com/istio/istio/blob/master/istioctl/cmd/sysexits.go

Reply:
I think that it makes sense. But we should mixer's and sidecar-injector's probe command also has same rule, so perhaps it would be better to change in separated PR?